toolsrc/src/vcpkg/commands.create.cpp

```diff
@@ -38,7 +39,9 @@ namespace vcpkg::Commands::Create
             {"CMD", "CREATE"},
             {"PORT", port_name},
             {"URL", url},
-            {"PORT_PATH", fs::generic_u8string(paths.builtin_ports_directory() / fs::u8path(port_name))}};
+            {"PORT_PATH", fs::generic_u8string(paths.builtin_ports_directory() / fs::u8path(port_name))},
+            {"VCPKG_BASE_VERSION", Commands::Version::base_version()},
```

Comment:
This is a minor nit, but
```suggestion
            {"VCPKG_BASE_VERSION", Commands::Version::base_version()},
            {"PORT_PATH", fs::generic_u8string(paths.builtin_ports_directory() / fs::u8path(port_name))},
```